pystan/model.py

```diff
@@ -554,7 +554,7 @@ def sampling(self, data=None, pars=None, chains=4, iter=2000,
 
         algorithm : {"NUTS", "HMC", "Fixed_param"}, optional
             One of algorithms that are implemented in Stan such as the No-U-Turn
-            sampler (NUTS, Hoffman and Gelman 2011), static HMC, or ``Fixed_param``.
+            sampler (Default, NUTS, Hoffman and Gelman 2011), static HMC, or ``Fixed_param``.
```

Comment:
Would you mind changing this to match how defaults are specified in the rest of the file?

e.g.

```

        algorithm : {"LBFGS", "BFGS", "Newton"}, optional
            Name of optimization algorithm to be used. Default is LBFGS.
```